src/Features/LanguageServer/Protocol/Features/CodeFixes/CodeFixService.cs

```diff
@@ -482,6 +508,14 @@ private bool TryGetWorkspaceFixersPriorityMap(TextDocument document, [NotNullWhe
 
                     foreach (var (span, diagnostics) in fixerToRangesAndDiagnostics[fixer])
                     {
+                        // Log an individual telemetry event for slow codefix computations to
+                        // allow targeted trace notifications for further investigation. 500 ms seemed like
+                        // a good value so as to not be too noisy, but if fired, indicates a potential
+                        // area requiring investigation.
+                        const int CodeFixTelemetryDelay = 500;
```

Comment:
1. can this be stored in a common location (this is the second time i've seen it)
2. can we switch this to a feature-flag.  That way if this is too noisy (or not noisy enough), we can remotely change it.

Reply:
From what I see, feature flags are just for flags, not for int values. I could use a feature flag to turn on/off this telemetry, but not to change the telemetry timing threshold.